### PR TITLE
Dev 1049 save metadata as collateral

### PR DESCRIPTION
**Pull request by RudolfDG**
This also fixes DEV-1134 and partly DEV-1069 (non-SUCCES events get dropped)

### CODE REVIEW

File: app/helpers/xml_helper.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    mdprops = etree.SubElement(root, "MDProperties")
+    etree.SubElement(mdprops, "CP").text = "VRT"
+    etree.SubElement(mdprops, "CP_id").text = "OR-rf5kf25"
+    etree.SubElement(mdprops, "sp_name").text = "s3"
+    etree.SubElement(mdprops, "PID").text = f"{pid}_metadata"
+    etree.SubElement(mdprops, "md5").text = metadata["Md5"]
+
+    relations = etree.SubElement(mdprops, "dc_relations")
+    etree.SubElement(relations, "is_deel_van").text = pid
```
Comment (maartends) [inline app/helpers/xml_helper.py:39]: See ticket DEV-1159 (this will be a new ticket).

File: app/services/ftp.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+from ftplib import FTP as BuiltinFTP
+from urllib.parse import urlparse
+
+# Third-party imports
+from viaa.configuration import ConfigParser
+from viaa.observability import logging
+from lxml import etree
+
+# Local imports
+import traceback
```
Comment (maartends) [inline app/services/ftp.py:13]: What is the purpose of this import?

**Review by maartends — APPROVED** [review]
- Nice seperation of `Event` and `Metadata`. These are indeed two different kinds of "things".
- Needs to be testied "in the wild" now... :-)

:+1:

File: app/helpers/events_parser.py [hunk 3]
```diff
@@ -12,26 +15,33 @@
...
-    def __init__(self, event_type: str, xml: bytes):
-        self.event = self._get_event(event_type, xml)
-        # TODO: Timestamp is currently optional, waiting on VRT to implement, see DEV-1052
-        self.timestamp = self._get_xpath_from_event("./vrt:timestamp", optional=True)
-        self.metadata = self._get_xpath_from_event("./vrt:metadata", xml=True)
-        self.media_id = self._get_xpath_from_event(
-            "//ebu:identifier[@typeDefinition='MEDIA_ID']/dc:identifier"
-        )
-        self._validate_metadata()
+            if status == "SUCCESS":
```
Comment (maartends) [inline app/helpers/events_parser.py:30]: Is a non-SUCCESS status begin logged (at least WARN)?

File: app/app.py [hunk 4]
```diff
@@ -90,7 +120,7 @@ def handle_message(self, channel, method, properties, body):
             channel.basic_nack(delivery_tag=method.delivery_tag, requeue=True)
             return
 
-        # 5. Update mediahaven fragement with received metadata
+        # 6. Update mediahaven fragement with received metadata
```
Comment (spacid) [inline app/app.py:123]: Minor typo: fragment

File: app/models/event.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
+from app.models.metadata import Metadata
```
Comment (spacid) [inline app/models/event.py:1]: Unused import

**Review by spacid — APPROVED** [review]

**Pull request merged**